media-libs/opencv/opencv-4.4.0.ebuild

```diff
@@ -0,0 +1,383 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6,7,8,9} )
+
+inherit java-pkg-opt-2 java-ant-2 python-r1 toolchain-funcs cmake-multilib
```

Comment:
Sort

Reply:
it turns out build will fail if cmake-multilib is put in front of everything else